docker/Makefile

```diff
@@ -18,7 +18,7 @@
 ###############################################################################
 # Parameters for this Makefile.
 BASE_IMAGE=seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete
-BASE_IMAGE_VERSION=v0.4.0
+BASE_IMAGE_VERSION=v0.5.0
```

Comment:
mmh by looking at the commit history in .core it looks like the changes were mostly about the Compact Point Cloud and IMU code, while in ODV